TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -15,7 +15,52 @@ public CodeElement CreateCommitStatement(CodeElementsParser.CommitStatementConte
         {
             return new CommitStatement();
         }
+        public CodeElement CreateTruncateStatement(CodeElementsParser.TruncateStatementContext context)
+        {
+            var tableName = CreateTableOrViewOrCorrelationName(context.tableName);
+            var storageManagementClause =
+                CreateStorageManagementClause(context.storageManagementClause());
+            var deleteTriggersHandlingClause = CreateDeleteTriggersHandlingClause(context.deleteTriggersHandlingClause());
+            var isImmediate = new SyntaxProperty<bool>(context.SQL_IMMEDIATE()!=null,null);
```

Comment:
No. By passing `null` in second argument you never capture the token so the usage of `SyntaxProperty` is useless.

```csharp
var isImmediate = context.SQL_IMMEDIATE() != null ? new SyntaxProperty<bool>(true, context.SQL_IMMEDIATE()) : null;
```

Reply:
Why you did not change here ? The code is currently wrong.